shell/src/main/java/alluxio/cli/fs/command/PersistCommand.java

```diff
@@ -68,6 +68,15 @@
               + DEFAULT_TIMEOUT)
           .required(false)
           .build();
+  private static final int DEFAULT_WAIT_TIME = 0;
+  private static final Option WAIT_OPTION =
+      Option.builder("w")
+          .longOpt("wait")
+          .argName("the initial persistence wait time")
+          .numberOfArgs(1)
+          .desc("The time in milliseconds to wait before persisting. default: " + DEFAULT_WAIT_TIME)
```

Comment:
Can we utilize the InstancedConfiguration's `getMs` function (I think it references `CommonUtils.Xyz` something) to parse values like `1min`, `1hour` etc instead of only supporting milliseconds?

Reply:
Change to allow given time formats. Thanks